src/layouts/EditContactUs.jsx

```diff
@@ -205,7 +205,7 @@ const EditContactUs = ({ match }) => {
     onClose: onDeleteModalClose,
   } = useDisclosure()
   const { data: contactUsDetails } = useGetContactUsHook(siteName)
-  const { data: settingsData } = useGetSettings(siteName)
+  const { data: settingsData } = useGetSettings(siteName, false) // Doesn't need the privatisation data, so we can always put this as a github login user
```

Comment:
sanity check here: this should be another todo right?

Reply:
Nope, this is a comment for clarity - we never need the privacy settings here, and `useGetSettings` does not retrieve privacy settings if the user is from github - this is just a small optimisation

Reply:
As discussed offline, we're changing the second argument to an optional value - https://github.com/isomerpages/isomercms-frontend/pull/1344/commits/83e780e3743cf8b573cf942a0ba576ddf03f316d